packages/react/src/modules/in-app-message/components/Modal/styles.css

```diff
@@ -0,0 +1,79 @@
+.knock-iam-modal {
```

Comment:
I'm pretty skeptical about the `-iam` bit, feels like it can be just `knock-modal`, but we can decide together later.

Reply:
The idea is to namespace the styles as there could be overlap between other features (like Slack kit). In this case, yeah it's unlikely we'd have another modal, but it is helpful in cases like `knock-iam-accent` where it's more likely we'd have an accent color in another component set.